api-report/tree.api.md

```diff
@@ -32,7 +33,25 @@ export abstract class BrandedType<ValueType, Name extends string> {
 }
 
 // @public (undocumented)
-export function buildForest(): IEditableForest;
+export function buildForest(schema: StoredSchemaRepository): IEditableForest;
```

Comment:
Document

Reply:
this is documented. API extractor lists things as undocumented here if the body of their doc comment is empty (not including the tags and inherited docs as far as I can tell). Personally, I feel the existing comment on this function (which is all in the `returns` tag) is sufficient, but let me know if you feel otherwise.

Reply:
As far as I can tell we only have API docs for 4 client packages: https://fluidframework.com/docs/apis/fluid-framework/ so I don't think we get any generated docs no matter what we do. That said, if api-documenter doesn't include docs from tags, that would be pretty broken / useless, and they work find in intelisense, so I'll assume its fine.

Reply:
I was actually just able to find it in our internal docs so your assumption would be correct 😄: https://eng.ms/docs/experiences-devices/oxo/office-shared/wacbohemia/fluid-framework-internal/fluid-framework/docs/apis/main/tree#buildforest-tree-buildforest-function

Reply:
Good find!